man1/aur.1

```diff
@@ -113,11 +113,19 @@ applicable.
 
 .SH ENVIRONMENT
 .TP
+.B AUR_DEBUG
+Enable debug mode (typically
+.B xtrace
+for
+.BR bash (1)
+scripts) in
+.B aur
+programs.
 
 .TP
 .B NO_COLOR
 Output colorization can be disabled by setting this environment
-variable to any value.
+variable to any non-empty value.
```

Comment:
This change is incorrect. Colored output will be disabled by just having the variable set, regardless of its value.

```suggestion
variable.
```